src/containers/App/index.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import type {AppProps, AppState} from './types';
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
+import {connect} from 'react-redux';
+import './App.css';
+import {Logger} from '../../services/logger';
+import Home from '../Home';
+import AppHeader from '../../components/AppHeader';
+import AppFooter from '../../components/AppFooter';
+
+export class App extends Component<AppProps, AppState> {
+  componentDidMount() {
+    const logger = new Logger();
+    logger.log('testing', 'Info');
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <AppHeader />
+        <Route exact path="/" component={Home} />
+        <AppFooter buildInformation={this.props.buildInformation} />
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = (state: AppState): AppProps => {
```

Comment:
this should be exported and added to test cases. Plus the state should be destructured to just fetch what we need from it, in this case we need `app`

```javascript
export const  mapStateToProps = ({ app: TypeofApp }): TypeofAppProps => ({ buildInformation: app.buildInformation })
```

Reply:
A test case for this`mapStateToProps` can be something like this:

```javascript
describe('mapStateToProps', () => {
  it('should return app buildinformation object', () => {
    expect(mapStateToProps({app: {buildInformation: {}}}).buildInformation).toEqual({})
  })
})
```